n3fit/src/n3fit/layers/preprocessing.py

```diff
@@ -5,11 +5,11 @@
 
 class Preprocessing(MetaLayer):
     """
-    Applies preprocessing to the PDF.
+    Computes prefactor for the PDF.
 
     This layer generates a factor (1-x)^beta*x^(1-alpha) where both beta and alpha
-    are model paramters that can be trained. If feature scaling is used, the preprocessing
-    factor is x^(1-alpha).
+    are model paramters that can be trained. If feature scaling is used, the
+    prefactor is x^(1-alpha).
```

Comment:
Please leave "preprocessing" in the documentation since this is the notation for this quantity that we have been using in papers [example here](https://arxiv.org/pdf/2109.02653.pdf).

Reply:
Oh I thought I had reverted all those changes. I turned preprocessing into preprocessing factor if it wasn't already there, just to distinguish it from things like feature scaling (only in documentation), hope that's ok.

Reply:
Sure, preprocessing factor is fine. But the word preprocessing should remain.

Indeed, in the physics (as in, when we discuss the physics in the paper) both this factor and the feature scaling are forms of preprocessing.